Options ignored on the command line

On Windows, Mambaforge Python 3.9.7 environment:

```
perflint mypackage/ --disable=loop-invariant-global-usage
...
mypackage\__main__.py:122:65: W8202: Lookups of global names within a loop is inefficient, copy to a local variable outside of the loop first. (loop-invariant-global-usage
```

Same with `-d W8202`.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -3,6 +3,7 @@
 ## 0.8.0 (main)
 
 * Bugfix: No longer suggests list comprehensions when the if statement has an elif/else clause
+* Bugfix: Pylint enable/disable arguments are allowed on the `perflint` CLI entry point
 
 ## 0.7.0 (28th March 2022)
 

--- a/perflint/__main__.py
+++ b/perflint/__main__.py
@@ -16,10 +16,12 @@ rules = (
     + list(ComprehensionChecker.msgs.keys())
 )
 
-args = sys.argv[1:]
+args = []
 args.append("--load-plugins=perflint")
 args.append("--disable=all")
 args.append("--enable={0}".format(",".join(rules)))
+args.extend(sys.argv[1:])
+
 try:
     PylintRun(args)
 except KeyboardInterrupt:

